stable/index.yaml

```diff
@@ -3,7 +3,7 @@ entries:
   nginx-appprotect-dos-arbitrator:
   - apiVersion: v1
     appVersion: 1.1.0
-    created: "2023-03-28T22:42:48.084553+01:00"
+    created: "2023-04-05T14:59:26.973352-07:00"
```

Comment:
Updating all time stamps seems wrong. This takes away history of when other files were created. This doesn't seem normal?

Reply:
we can fix it in the upcomming updates, im not sure why helm is doing that